Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM frolvlad/alpine-python3
```

Comment:
Is there a reason you've used this rather than just installing python3 using apk?